core/trino-main/src/main/java/io/trino/operator/aggregation/AggregationConfig.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator.aggregation;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.LegacyConfig;
+import io.airlift.units.DataSize;
+
+public class AggregationConfig
```

Comment:
See https://github.com/trinodb/trino/pull/11066#discussion_r817641251. We should extract `OptimizerConfig` config file. I suggest you skip this extract for now since it will be part of https://github.com/trinodb/trino/pull/11066